rails/tasks/whenever/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 - name: Update cronjobs for app
-  command: "{{ profile }}'bundle exec whenever --update-crontab {{ app_name }}'"
+  command: "{{ profile }}'bundle exec whenever --set environment={{ rails_env}} --update-crontab {{ app_name }}'"
```

Comment:
`--set environment=$RAILS_ENV` should work here.

We never use the `rails_env` argument in the Rails roles. So it would be undocumented.